### ISSUE TITLE
Automatically set the hyperparameter alpha in Xu's learning rate

### ISSUE THREAD

**Issue by dustinvtran** [airoldilab/sgd#55]
The previous automated approach was based on using eigenvalues of the observed Fisher information. This is both slow and leads to numerical blowups, c.f., #41.

**Comment by dustinvtran** [comment]
Marking this as duplicate given #17
